### ISSUE TITLE
Wrong Input type in frame_rate (Video Combine 🎥🅥🅗🅢 Node)

### ISSUE THREAD

**Issue by GoldenCrowInf** [Kosinkadink/ComfyUI-VideoHelperSuite#234]
Hello,

I'm having this problem, I've set the frame_rate from Video Combine 🎥🅥🅗🅢 to Widget to Input. When I do it, the required input value is an INT. When I run the workflow, it returns an error saying that the frame_rate input is not a FLOAT.

* (prompt):
  - Return type mismatch between linked nodes: frame_rate, INT != FLOAT
* VHS_VideoCombine 193:
  - Return type mismatch between linked nodes: frame_rate, INT != FLOAT
Output will be ignored

In previous versions it was working fine, but now it's broken.

I hope it helps!

**Comment by AustinMroz** [comment]
The frame_rate input was changed to float back in March for compatibility with the frame rate values produced from Load Video/Video Info. If you create a new Video Combine node and convert frame_rate to an input, is it still of type int for you?

**Comment by GoldenCrowInf** [comment]
Yes, weh you turn frame_rate into input, it only accepts INT as input value. this is the problem. If I convert to float, it cannot take the value as input. You cannot connect it as float.
![Captura de tela 2024-06-19 173845](https://github.com/Kosinkadink/ComfyUI-VideoHelperSuite/assets/121122993/a5db56fa-1516-4b09-b0c0-c48bbd7cbfdc)

**Comment by GoldenCrowInf** [comment]
Ok. Nevermind. I was loading an old Workflow. And in this workflow it was using an old version of Video Combine. I made a new workflow and it's working fine. Sorry about that and thank you!
